SPECS-EXTENDED/bolt/bolt.spec

```diff
@@ -0,0 +1,167 @@
+Summary:        Thunderbolt device manager
+Name:           bolt
+Version:        0.9.2
+Release:        2%{?dist}
+License:        LGPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://gitlab.freedesktop.org/bolt/bolt
+Source0:        %{url}/-/archive/%{version}/%{name}-%{version}.tar.gz
+BuildRequires:  asciidoc
+BuildRequires:  gcc
+BuildRequires:  libudev-devel
+BuildRequires:  meson
+BuildRequires:  pkgconfig
+BuildRequires:  polkit-devel
+BuildRequires:  systemd
+BuildRequires:  systemd-devel
+BuildRequires:  pkgconfig(gio-2.0)
+%{?systemd_requires}
+# for the integration test (optional)
+%if 0%{?fedora} || 0%{?rhel} > 8
+BuildRequires:  pygobject3-devel
+BuildRequires:  python3-dbus
+BuildRequires:  python3-dbusmock
+BuildRequires:  umockdev-devel
+%endif
```

Comment:
Can we double check to see if these deps make sense for us.

I think our preference is to not have `%if fedora/rhel` if possible.

Reply:
Thanks. I have removed the above block of code. Ran Buddy builds after that. Here are the results. 
- [AMD64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=305821&view=results)
- [ARM64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=305822&view=results)